plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs

```diff
@@ -308,8 +352,10 @@ private FileModel GenerateNewFileModel(FileModel model, PageViewModel newPage, s
             var initialFile = model.FileAndType.File;
             var extension = Path.GetExtension(initialFile);
             var directory = Path.GetDirectoryName(initialFile);
-            var newFileName = PathUtility.ToValidFilePath(key, '-').Replace('`', '-') + extension;
```

Comment:
seems no need to. 

---
In reply to: [102860136](https://github.com/dotnet/docfx/pull/1341#discussion_r102860136) [](ancestors = 102860136)